intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/EditorWindowListener.java

```diff
@@ -13,44 +19,53 @@
 
```

Comment:
Why these changes?

Reply:
They were part of my preparation for the ASAT functionality and to make it more idiomatic IntelliJ code. Will put them in a separate PR :+1: